app/lib/actions/pulp/orchestration/repository/refresh_if_needed.rb

```diff
@@ -3,7 +3,7 @@ module Pulp
     module Orchestration
       module Repository
         class RefreshIfNeeded < Pulp::Abstract
```

Comment:
Can you see if this action can be removed?. Kinda seems useless ?

Reply:
so this is kind of an 'adapter' to make it work with the pulp selector.  I could go through and re-work all instances of calls to Refresh, but there's some risk to that, and this will all be deleted when we drop pulp2 anyways.

Reply:
Hmm I see it used in only one place in the code base
```bash

$ git grep RefreshIfNeeded

app/lib/actions/katello/repository/clone_to_environment.rb:              plan_optional_pulp_action([Actions::Pulp::Orchestration::Repository::RefreshIfNeeded], clone, SmartProxy.pulp_master)
app/lib/actions/pulp/orchestration/repository/refresh_if_needed.rb:        class RefreshIfNeeded < Pulp::Abstract
[paji@sparta katello{master}]$ 

```

Reply:
Ok you mentioned you were handling this in a different PR. So resolving